server/odc-service/src/main/java/com/oceanbase/odc/service/task/listener/DefaultJobTerminateListener.java

```diff
@@ -54,5 +59,25 @@ public void onEvent(JobTerminateEvent event) {
                 }
             });
         }
+
+        Optional<ScheduleTaskEntity> scheduleTask = taskRepository.findByJobId(jobEntity.getId());
```

Comment:
add a unique key for this column

Reply:
add `public Optional<ScheduleTaskEntity> findByJobId(Long jobId) ` for check unique key